modules/cloudbuild/cloudbuild_builder/Dockerfile

```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/cloud-builders/gcloud-slim
+ARG GCLOUD_VERSION=slim
+FROM google/cloud-sdk:${GCLOUD_VERSION}
```

Comment:
We should allow overriding this via TF like how we do for `TERRAFORM_VERSION`

Reply:
Added variables into the tf modules